wiki/voting.md

```diff
@@ -0,0 +1,551 @@
+---
+title: Gridcoin Voting System
+layout: wiki
+description: Overview of Gridcoin's voting system, how to vote, what the requirements for polls are and more
+---
+
+# Introduction
+
+Gridcoin has a voting system in place to help find what the community thinks on
+a specific issue or where the community wants the coin to go. Polls on Gridcoin
+have ranged from [adding projects to the whitelist](whitelist-process#whitelist-poll "wikilink") to
+changing how rewards are given out
+
+# Voting on a Poll
+
+First, find the polls that are currently running. In the GUI this can be done by 
+clicking on the vote tab. Right click on the poll and press either vote or
+show results to see more information including the discussion about the poll.
+For non-GUI users, the `listpolls` command can be used to find active polls
+
+Make sure to read about the poll and see some of the discussion before you vote
+
+To vote, right click on the poll and press vote. Then click on your answer and
+press the vote button below. Non-GUI users can vote by using the `votebyid`
+command like this `votebyid <ID> <CHOICE>` (the poll ID can be found with the earlier
+mentioned `listpoll` command)
+
+
+# How Votes Are Weighted
+
+How much your vote counts is weighted based off of your balance plus (for solo crunchers) 
+your [magnitude](magnitude "wikilink").
+
+The formula for exactly how it's weighted can be found [here](#11-vote-weight-and-active-vote-weight "wikilink")
+
+# Poll Requirements
+In 2019, the current standards that polls need to follow was approved. It laid out
+what types of polls there were and what each poll type needs to pass 
+
+
+All polls but the opinion/casual polls require:
+* A main discussion thread on GitHub or reddit
+* Secondary discussion threads on GitHub, Reddit, and Steemit with links to each
+in each thread
+* Unbiased poll phrasing and options
+* An abstain option
+
+Each category has its own requirements which are listed in the [poll types](#poll-types "wikilink") section of this page 
+
+To pass, a poll must get a majority of the vote and must be at or above the poll's
+active vote weight (AVW) requirement. AVW is an estimate how much voting
+weight is from users who are active on the network. It is described in more detail 
+in the [active vote weight](#active-vote-weight "wikilink") section
+
+
+## Requirements for Funding
+Any poll that requests funding must state:
+* Intended use of funds
+* The date of work begin
+* The date of work end
```

Comment:
awkward wording

Reply:
Took that directly from the polls requirement doc, but agree that the wording there is awkward (should have looked closer at it). Changed it to 
```
Any poll that requests funding must state:
* The intended use of funds
* The date that the work begins (or began)
* The date that the work ends (or ended)
* Total amount of funding requested (in GRC)
* When funds are released
* How funds are released
* The payee's name or alias
* The payee's wallet address
* Reliable contact information
```